server/api/pkg/workflow/workflow.go

```
@@ -1,18 +1,40 @@
 
 type Workflow struct {
-	ID        ID `json:"id"`
-	Project   ProjectID
-	Workspace WorkspaceID
-	// Meta *string
-	URL string
+	id        ID
+	project   ProjectID
+	workspace WorkspaceID
+	// Meta      *string
+	url       string
```

golangci-lint (1.62.2)

File is not `goimports`-ed

(goimports)